src/styles/main.scss

```diff
@@ -1 +1,57 @@
-// put code here
+.body {
+  width: 100%;
+  height: 100vh;
+  display: grid;
+  place-items: center;
+  background-color: black;
+}
+
+.stopwatch {
+  position: relative;
+  width: 500px;
+  height: 500px;
+  border: 15px dotted darkred;
+  border-radius: 50%;
+
+  &__center-button {
+    position: absolute;
+    top: 235px;
+    left: 235px;
```

Comment:
it is better to use relative points of measure to center this element. You can make it like this:

```scss
top: 50%;
left: 50%;
transform: translate(-50%, -50%);
```